src/screenviewing/deltarenderer/DefaultScreenViewingDeltaRenderer.ts

```diff
@@ -188,6 +188,18 @@ export default class DefaultScreenViewingDeltaRenderer implements ScreenViewingD
     }
   }
 
+  hideViewport(): void {
+    if (this.viewport) {
+      this.viewport.style.display = 'none';
```

Comment:
I don't think we should modify the CSS styling of the viewport since this is an element that is passed in by the SDK consumer. In particular, they may be controlling the `display` attribute themselves and so this could overwrite the attribute. Is there some other way to accomplish the objective here without using CSS styling?

Reply:
As discussed offline, I modified the CSS for the canvas element rather than the exposed viewport.